cms/templates/js/certificate-details.underscore

```diff
@@ -0,0 +1,40 @@
+<div class="collection-details wrapper-certificate">
+    <header class="collection-header">
+        <h3 class="title">
+            <a href="#" class="toggle detail-toggle <% if (showDetails){ print('hide'); } else { print('show'); } %>-details">
```

Comment:
```href="#"``` is actually not valid.  It's usually a sure sign that this should be a ```<button>``` and not a hyperlink.  (Buttons do something, Links go somewhere).  I would make this a button, which means it probably shouldn't be wrapped in an h3.